core/lib/zksync_core/src/l1_gas_price/gas_adjuster/mod.rs

```diff
@@ -63,11 +86,11 @@ impl GasAdjuster {
             .as_usize()
             .saturating_sub(1);
 
-        let last_processed_block = self.statistics.last_processed_block();
+        let last_processed_block = self.base_fee_statistics.last_processed_block();
 
         if current_block > last_processed_block {
             // Report the current price to be gathered by the statistics module.
-            let history = self
+            let base_fee_history = self
```

Comment:
do we really need to call this function? we call N blocks for blocks no matter what.. should we just reuse info from blobs requestgs?

Reply:
You are right, we can remove it here. But I'd still leave calling feeHistory in constructor